docs/platform/release-notes/tizen-7-0-m1.md

```diff
@@ -0,0 +1,545 @@
+
+- Open source
+  - Wayland has been upgraded to version 1.20.0.
+  - Libxkb-common has been upgraded to version 1.3.1.
+  - Libinput has been upgraded to version 1.17.0.
+  - Libevdev has been upgraded to version 1.11.0.
+  - Mesa has been upgraded to version 21.2.3.
+- Enlightenment
+  - Drag and Drop has been added. It provides wl_data_device_manager interface of wayland protocol to the wayland-clients.
+- Vulkan
+  - Vulkan-wsi-layer has been added. It implements Vulkan window system integration extensions.
+- OpenCL
+  - Common OpenCL Packages have been added.
+    - OpenCL-Headers have been added. It contains C language headers for the OpenCL API.
+    - OpenCL-ICD-Loader has been added. It allows developers to build application against an Installable Client Driver loader.
+- TTS Framework
+  - In order to reduce memory usage, the TTS engine, which used to execute in multiple processes, has been improved to execute as one process.
+  - The client structure and TTS player to support thread safety has been improved.
+  - The TTS engine API to terminate the process has been added.
+  - The API to connect TTS engine synchronously has been added.
+  - The API to repeat TTS text has been added.
+- Sticker Framework
+  - C# sticker API has been added.
+- Text Input Framework
+  - Vietnamese and Thai language support has been added in the Tizen reference IME.
+  - The inputmethod C# API set to set preedit cursor position and set the callback to get input hint has been added.
+  - Wayland inputmethod interface has been updated as the latest open source.
+
+#### Fixes
+
+- Text Input Framework
+  - C# inputmethod API set returns the appropriate error exception.
+
+
+### Graphics and UI
+
+#### New and changed features
+
+- Rendering
+  - An interface to support direct rendering has been added.
+- Input and Gesture
+  - Support for CustomWheel Event has been added to the View.
+  - Some properties and signals regarding input and gesture have been added.
+- Window
+  - A method to receive an auxiliary message has been added.
+  - Methods to maximize and minimize a window have been added.
+  - Support for Drag and Drop has been added.
+- View
+  - Various methods have been added to the WebView.
+  - Various properties and methods have been added to the TextLabel, TextField and TextEditor.
+- ATSPI
+  - Multi window activate/deactivate has been added.
+  - TextField password information protection has been added.
+  - Hyperlink ATSPI Interface has been added.
+  - Performance to enable AT-SPI has been improved.
+  - MBR (Minimum Bounding Rectangle) has been added.
+  - Multiple AT-SPI interfaces for View have been added.
+  - Default label Feature has been added.
+  - NUI Component default accessibility has been added.
+  - Singleton Accessibility Delegate to enhance memory usage has been added.
+  - Text Control Accessible has been added.
+  - ATSPI/Screen-Reader Enabled/Disabled Signal has been added.
+  - ATSPI suppressed events have been added.

```

Comment:
@wtprince In this section, both ATSPI and AT-SPI are being used. I think it would be better to use either one of ATSPI or AT-SPI. What do you think?

Reply:
agree. let's use ATSPI.

Reply:
```suggestion
- ATSPI
  - Multi window activate/deactivate has been added.
  - TextField password information protection has been added.
  - Hyperlink ATSPI Interface has been added.
  - Performance to enable ATSPI has been improved.
  - MBR (Minimum Bounding Rectangle) has been added.
  - Multiple ATSPI interfaces for View have been added.
  - Default label Feature has been added.
  - NUI Component default accessibility has been added.
  - Singleton Accessibility Delegate to enhance memory usage has been added.
  - Text Control Accessible has been added.
  - ATSPI/Screen-Reader Enabled/Disabled Signal has been added.
  - ATSPI suppressed events have been added.

```

Reply:
will accept